Package@swift-5.7.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.6
+// swift-tools-version:5.7
```

Comment:
Same question.

Reply:
This one isn't needed, I just thought I'd align the Swift version with the swift-tools version. Is there a reason why you think this shouldn't change?